### DISCUSSION TITLE
Is it possible to save encoded file in the same directory as source file?

### DISCUSSION THREAD

**Discussion by mblandr** [Anime4000/IFME#246] [Q&A]
Is it possible to save encoded file in the same directory as source file?

**Comment by Anime4000** [comment]
if you change output path, yes, but I strongly avoid do that 😅

**Comment by mblandr** [comment]
No, I mean for every file in the list, paths may be different for one file to another. Not a fixed path for every file

**Comment by Anime4000** [comment]
I think I can make it, I try it first

**Reply by mblandr** [reply]
And may be transfer date attributes (created, modified) from source file to encoded?
If IFME will have two these features, it will be the best overall encoder!

**Reply by Anime4000** [reply]
`created, modified` date of file or as Metadata ?

**Reply by mblandr** [reply]
I don't know exactly, I mean Created, Modified on General Tab in the Properties Dialog for the file

**Reply by Anime4000** [reply]
you mean this?
![image](https://github.com/user-attachments/assets/3d42745e-f347-4de0-a901-5e95cf5bd942)

**Reply by mblandr** [reply]
yes
